tests/src/stdsystem.test.cpp

```diff
@@ -21,20 +21,25 @@ TEST_CASE_FIXTURE(CliRuntimeFixture, "std_system_os_matches_host_os")
 {
     runCode(R"(
         local system = require("@std/system")
-        capture(system.os)
+        report(system.os)
     )");
-    CHECK(getCapturedOutput() == getHostOS());
+    CHECK(getReporter().getOutputs()[0] == getHostOS());
 }
 
 TEST_CASE_FIXTURE(CliRuntimeFixture, "check_std_system_env_bools")
 {
     std::string os = getHostOS();
 
-    auto checkBool = [&](const std::string& field, bool expected) {
-        runCode("local system = require(\"@std/system\")\n"
-                "capture(system." + field + ")\n");
-        std::string output = getCapturedOutput();
+    auto checkBool = [this](const std::string& field, bool expected)
+    {
+        std::string code = R"(
+            local system = require("@std/system")
+            report(system.)" + field + R"()
+        )";
```

Comment:
Sorry, I didn't realize that `field` was being added in the middle of this; my raw string suggestion is also confusing. Maybe `Luau::format` would be useful here?
```suggestion
        std::string code = Luau::format(R"(
            local system = require("@std/system")
            report(system.%s)
        )", field.c_str());
```